### PR TITLE
more straightforward conditional

### PR DESCRIPTION
**Pull request by chipitsine**
we actually do not need so many "goto"s

### CODE REVIEW

File: windows-nsis/openvpn.nsi [hunk 1]
```diff
@@ -475,10 +472,7 @@ Function .onInit
 
 ${IfNot} ${AtLeastWinVista}
 
-	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDNO guiEndNo
-	Goto guiEndYes
-
-	guiEndNo:
+	MessageBox MB_YESNO|MB_ICONEXCLAMATION "This package does not work on your operating system.  The last version of OpenVPN supported on your OS is 2.3. Shall I open a web browser for you to download it?" /SD IDYES IDYES guiEndYes
```
Comment (selvanair) [inline windows-nsis/openvpn.nsi]: /SD IDYES should be /SD IDNO so that the browser wont popup in silent installs.

File: windows-nsis/openvpn.nsi [hunk 2]
```diff
@@ -475,10 +472,7 @@ Function .onInit
...
 	Quit
 
 	guiEndYes:
```
Comment (selvanair) [inline windows-nsis/openvpn.nsi]: While at it please also change this label to something more meaningful like WinXPYes as @mattock had pointed out earlier.

**Comment by selvanair** [comment]
While this is good, let's refrain from beautifying PRs until 2.4 is out. Every PR takes time to review and merge and we have a number of critical changes pending.

**Comment by chipitsine** [comment]
sure, it is not critical

**Comment by selvanair** [comment]
If you can address my comments I can test it out -- now that we have started staring at it, better complete it :)

**Comment by chipitsine** [comment]
I pushed new commit, can you test it ?

**Comment by selvanair** [comment]
All good except I cannot test WinXP install message box (no XP at hand). But no reason to expect any issues with it.

**Comment by mattock** [comment]
This PR partially overlaps with PR https://github.com/OpenVPN/openvpn-build/pull/62 which I just created, before noticing this PR. I can rip out the [WinXP label commit](https://github.com/OpenVPN/openvpn-build/pull/62/commits/ce5876b9cee2358ae02347183f212674cfcf2945) from my PR if this approach is seen as better.

**Comment by chipitsine** [comment]
@selvanair also noticed that "/SD IDYES" is bad approach.

**Comment by chipitsine** [comment]
I do not mind, if you merge your own PR, please close this one

**Comment by selvanair** [comment]
@chipitsine : we still need this because of the /SD IDNO change apart from the cleaning up. To avoid conflict with PR 62 you may have to remove the label change and rebase after that one is merged (or the other way). Otherwise ACK from me.

**Comment by chipitsine** [comment]
ok, do whatever you think is proper

**Comment by chipitsine** [comment]
this PR#61, is cannot conflict with #62 
#62 is _later_ PR. I do not understand your point that my PR conflict.

I think proper idea is that 62 PR conflicts and its author shall take care of it

**Comment by selvanair** [comment]
Hi,

> this PR#61, is cannot conflict with #62
> <https://github.com/OpenVPN/openvpn-build/pull/62>
> #62 <https://github.com/OpenVPN/openvpn-build/pull/62> is *later* PR. I
> do not understand your point that my PR conflict.
>
> I think proper idea is that 62 PR conflicts and its author shall take care
> of it
>

It depends on which is merged first. Anyway let's wait for @mattock to act.

Selva

**Comment by mattock** [comment]
@chipitsine can you rebase this to latest "master"? Afaics the changes still make sense, but merging of PR https://github.com/OpenVPN/openvpn-build/pull/62 has caused merge conflicts.

**Comment by chipitsine** [comment]
I rebased to the latest "master"

https://github.com/OpenVPN/openvpn-build/pull/81

**Pull request closed without merging**